README.md

```diff
@@ -1,61 +1,29 @@
 # lco-education-platform
 
-## Project setup
-```
-npm install
-```
-
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
-
-### Compiles and minifies for production
-```
-npm run build
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
-
-## Running this project locally via Kubernetes
-
-This project includes kubernetes manifests that allow it to be deployed to a kubernetes cluster.
+This application is a part of the Photon Ranch Collection along with DataLab and the Education Platform. 
+Observe@PTR is where users, typically students, will go to request observations on the 40cm network and perform live observing.  
+![Screenshot of Dashboard](./Dashboard.png)
 
-To run this project in a local kind cluster:
+## Project Setup Instructions
 
-Drop into the nix environment that contains all of the relevant tools.
+## Getting Started Locally
+### STEP 1: Install Nix
+If you don't have Nix installed, you can follow [these steps](https://github.com/LCOGT/public-wiki/wiki/Install-Nix)
+### STEP 2: Enter the Nix Dev Environment
 ```
 nix develop --impure
 ```
-
-Create a cluster and container registry.
+### STEP 3: Apply Configurations
 ```
-ctlptl apply -f local-cluster.yaml -f local-registry.yaml
+ctlptl apply -f local-registry.yaml local-cluster.yaml
 ```
 
-Make sure you update the skaffold.env.changeme and k8s/base/secrets.env.changeme to include the fontawesome package token
-
-```
-mv skaffold.env.changeme skaffold.env
-mv k8s/base/secrets.env.changeme k8s/base/secrets.env
-```
+**NOTE**: If when trying to apply the configuration you see this error `Nothing found at DockerDesktop setting "vm"` and are using Docker Desktop, using OrbStack should fix that issue. This may also be an M2 problem (Apple Silicon). Or update `ctlptl` (more notes to add here after Friday)
```

Comment:
So I think once the nix flake in this repo is updated with Jashan's upstream changes, this note shouldn't be necessary - so maybe that update should be part of the PR.